tracee-rules/signature.go

```diff
@@ -44,6 +44,7 @@ func getSignatures(rulesDir string, rules []string) ([]types.Signature, error) {
 	return res, nil
 }
 
+// TODO: Use filepath.Walk() to recursively scan
```

Comment:
please create an issue for this feature request and remove this from the code.. (same for the other similar comment)

Reply:
Roger https://github.com/aquasecurity/tracee/issues/501